backend/lambdas/api/spec.yaml

```diff
@@ -3145,6 +3195,98 @@ components:
               type: integer
               format: int32
 
+
+    #System Service
+    SystemService:
+      type: object
+      required:
+        - service
+        - reachable
+        - auth_successful
+        - auth_type
+        - error
+      properties:
+        service:
+          description: Service name
+          type: string
+        reachable:
+          description: Whether the service is reachable
+          type: boolean
+        auth_successful:
+          description: Whether Artemis can successfully authenticate to the service
+          type: boolean
+        auth_type:
+          description: Type of authentication Artemis uses to connect to the service
+          type: string
+        error:
+          description: Error encountered when testing connection from Artemis to service
+          type: string
+          nullable: true
+    
+    SystemServiceStats:
+      type: object
+      required:
+        - service
+        - repo_count
+        - total_scans
+        - successful_scans
+        - failed_scans
+        - first_scan
+        - most_recent_scan
+      properties:
+        service:
+          description: Service name
+          type: string
+        repo_count:
+          description: Count of repos scanned in the service
+          type: integer
+          format: int32
+        total_scans:
+          description: Count of all scans run on repos in this service
+          type: integer
+          format: int32
+        successful_scans:
+          description: Count of successful scans run on repos in this service
+          type: integer
+          format: int32
+        failed_scans:
+          description: Count of failed scans run on repos in this service
+          type: integer
+          format: int32
+        first_scan:
+          description: Timestamp of earliest scan run on a repo in this service
+          type: string
+          format: date-time
+        most_recent_scan:
+          description: Timestamp of most recent scan run on a repo in this service
+          type: string
+          format: date-time
```

Comment:
I think I prefer the `timestamps` block too, to keep it consistent with a scan result.
Instead of `most_recent`, I'd prefer `latest_` or `last_` (we use `last_` in other places, such as last qualified scan)

Reply:
done, implemented as:
```
"timestamps": {
  "oldest_scan": "<timestamp>",
  "latest_scan": "<timestamp>"
}
```